spec/policy_chain_spec.lua

```diff
@@ -92,6 +92,38 @@ describe('policy_chain', function()
     end, 'readonly table')
   end)
 
+  describe('.insert', function()
+
+    it('adds policy to the chain', function()
+      local chain = _M.new({})
+
+      chain:insert(policy, 1)
+
+      assert.equal(policy, chain[1])
+      assert.equal(1, #chain)
+    end)
+
+    it('adds a policy to specific position', function()
+      local chain = _M.new({ 'one', 'two'})
+
+      chain:insert(policy, 2)
+      assert.equal(policy, chain[2])
+      assert.equal('one', chain[1])
+      assert.equal('two', chain[3])
+      assert.equal(3, #chain)
+    end)
+
+    it('errors when inserting to frozen chain', function()
+      local chain = _M.new({}):freeze()
+
+      local ok, err = chain:insert(policy, 1)
+
+      assert.is_nil(ok)
+      assert.equal(err, 'frozen chain')
+      assert.equal(0, #chain)
+    end)
+  end)
+
```

Comment:
Would be good to test `.default` for completeness. The other two methods introduced in this PR for that module are tested.

Reply:
Good point. Fixing 👍

Reply:
Fixed in https://github.com/3scale/apicast/pull/496/commits/1fbfff80566e9728a6a77b5cab1db2b6d00a3a5b and will merge it in a rebase.